browser/src/Editor/NeovimEditor/BufferLayerManager.ts

```diff
@@ -69,6 +69,13 @@ export class BufferLayerManager {
     }
 }
 
+const getInstance = (() => {
```

Comment:
Just curious - why is this wrapped in an IIFE? Is there a functionaility reason for it?

Reply:
It's to encapsulate the instance inside of the function and just return a function that provides access to it,

So rather than
```ts
	let instance = new BufferLayerManager() // global variable
	export getInstance = () => instance

	const getInstance = (() => {
		const instance = new Manager() // scoped to the IIFE and accessed via the closure
		return () => {
			return instance
		}
```

Its a very small distinction I've just been bitten by global vars previously and try to always find ways around them.